commit 21bbc03c0611a2f2920ed684e7586f795a5486c8
Author: Jaegeuk Kim <jaegeuk@kernel.org>
Date:   Tue Feb 14 09:54:37 2017 -0800

    f2fs: fix multiple f2fs_add_link() calls having same name
    
    commit 88c5c13a5027b36d914536fdba23f069d7067204 upstream.
    
    It turns out a stakable filesystem like sdcardfs in AOSP can trigger multiple
    vfs_create() to lower filesystem. In that case, f2fs will add multiple dentries
    having same name which breaks filesystem consistency.
    
    Until upper layer fixes, let's work around by f2fs, which shows actually not
    much performance regression.
    
    Signed-off-by: Jaegeuk Kim <jaegeuk@kernel.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1339_21bbc03c0611a2f2920ed684e7586f795a5486c8.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1339_21bbc03c0611a2f2920ed684e7586f795a5486c8.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
